runhouse/main.py

```diff
@@ -590,159 +592,70 @@ def status(
+    ),
+    since: Optional[str] = typer.Option(
+        None,
+        "--since",
+        help="Time duration to filter on. Minimum allowable filter is 1 minute. You may filter by seconds (s), "
+        "minutes (m), hours (h) or days (s). Examples: 30s, 15m, 2h, 3d.",
+    ),
+    cluster_status: Optional[ClustersListStatus] = typer.Option(
+        None,
+        "--status",
+        help="Cluster status to filter on. Supported filter values: running, terminated (cluster is not live), down (runhouser server is down, but the cluster might be live).",
+    ),
+):
+    """Load Runhouse clusters"""
 
     # logged out case
     if not rh.configs.token:
         # TODO [SB]: adjust msg formatting (coloring etc)
         sky_cli_command_formatted = f"{italic_bold_ansi}sky status -r{reset_format}"  # will be printed bold and italic
         console.print(
-            f"Runhouse token required to view all clusters. Please run `runhouse login` to load your token. To view on-demand clusters run {sky_cli_command_formatted}"
+            f"Listing clusters requires a Runhouse token. Please run  `runhouse login` to get your token, or run {sky_cli_command_formatted} to list locally stored on-demand clusters."
         )
         return
 
-    sky_clusters = sky.status()
-
-    clusters_in_den_resp = get_clusters_from_den()
-
-    if clusters_in_den_resp.status_code != 200:
-        logger.error(
-            f"Failed to load clusters from Den for username: {rns_client.username}"
-        )
-        clusters_in_den = []
-    else:
-        clusters_in_den = clusters_in_den_resp.json().get("data")
+    clusters = Cluster.list(show_all=show_all, since=since, status=cluster_status)
 
-    clusters_in_den_names = [cluster.get("name") for cluster in clusters_in_den]
+    all_clusters = clusters.get("all_clusters", None)
+    running_clusters = clusters.get("running_clusters", None)
+    sky_clusters = clusters.get("sky_clusters", None)
 
-    if not sky_clusters and not clusters_in_den:
-        console.print("No existing clusters.")
-
-    if sky_clusters:
-        # getting the on-demand clusters that are not saved in den.
-        sky_clusters = [
-            cluster
-            for cluster in sky_clusters
-            if f'/{rns_client.username}/{cluster.get("name")}'
-            not in clusters_in_den_names
-        ]
-
-    total_clusters = len(clusters_in_den)
-    table_title = f"[bold cyan]Clusters for {rns_client.username} (Total: {total_clusters})[/bold cyan]"
-
-    table = Table(title=table_title, title_justify="left")
-
-    # Add columns to the table
-    table.add_column("Name", justify="left", no_wrap=True)
-    table.add_column("Cluster Type", justify="center", no_wrap=True)
-    table.add_column("Status", justify="left")
-
-    running_clusters = []
-    not_running_clusters = []
+    if not all_clusters:
+        console.print(f"No clusters found in Den for {rns_client.username}.")
```

Comment:
should we make this statement more dynamic? for example if i provide some filter that leads to no results, doesn't necessarily mean there were no clusters found in Den